cpufreq/src/Makefile.am

```diff
@@ -35,7 +38,10 @@ mate_cpufreq_applet_CFLAGS =		\
 mate_cpufreq_applet_LDADD =		\
 	$(MATE_APPLETS4_LIBS)   	\
 	$(MATEDESKTOP_LIBS)		\
-	$(LIBCPUFREQ_LIBS)		\
-	$(DBUS_LIBS)
+	$(LIBCPUFREQ_LIBS)
+
+if HAVE_POLKIT
+mate_cpufreq_applet_LDADD += $(POLKIT_LIBS)
```

Comment:
This now adds `polkit-gobject-1` library here but polkit it self is not used in applet.

Reply:
You're right, here it just needs to use DBUS LIBS & CFLAGS. I also removed MATEDESKTOP LIBS & CFLAGS. Test: `git grep MATEDESKTOP` , `grep MATEDESKTOP config.log`.

Reply:
DBUS LIBS & CFLAGS are no longer required in this file after porting to GDBus the clients of cpufreq-selector.